### DISCUSSION TITLE
Implemented: Accessible, spec-compliant Notification Component for Pico (using <dialog role="alert">)

### DISCUSSION THREAD

**Discussion by kyr0** [picocss/pico#552] [Ideas]
Hi all,

I've built a spec-compliant, accessible Notification Component for Pico, closely following [web standards for presentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) and [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role).

[Implementation and Demo of the Notification Component](https://codepen.io/kyr0/pen/abrpvRe)

Feel free to discuss/integrate it/take what you need. 
Permission hereby granted under the terms and conditions of the MIT license.

Implementation: #553 

Best,
Aron

**Comment by sysmat** [comment]
Nice
